ncm-afsclt/src/main/perl/afsclt.pm

```diff
@@ -25,55 +25,60 @@ Readonly my $THESECELLS    => '/usr/vice/etc/TheseCells';
 Readonly my $AFSD_ARGS     => '/etc/afsd.args';
 Readonly my $PREFIX        => '/software/components/afsclt';
 
+Readonly my $AFS_MOUNTPOINT_DEF => '/afs';
+
+
 sub Configure {
     my ( $self, $config ) = @_;
-    $self->Configure_Cell($config);
-    $self->Configure_TheseCells($config);
-    $self->Configure_Cache($config);
-    $self->Configure_CellServDB($config);
-    $self->Configure_Afsd_Args($config);
+
+    my $afsclt_config = $config->getElement($PREFIX)->getTree();
+
+    $self->Configure_Cell($afsclt_config);
+    $self->Configure_TheseCells($afsclt_config);
+    $self->Configure_Cache($afsclt_config);
+    $self->Configure_CellServDB($afsclt_config);
+    $self->Configure_Afsd_Args($afsclt_config);
 }
 
 sub Configure_Cell {
     my ( $self, $config ) = @_;
-    unless ( $config->elementExists("$PREFIX/thiscell") ) {
-        $self->error("Cannot get $PREFIX/thiscell (required in profile)");
+    unless ( defined($config->{thiscell}) ) {
+        $self->error("$PREFIX/thiscell missing in the configuration");
         return 1;
     }
 
-    my $afscell = $config->getValue("$PREFIX/thiscell");
     my $thiscell_fh = CAF::FileWriter->new( $THISCELL, log => $self );
-    print $thiscell_fh "$afscell\n";
+    print $thiscell_fh $config->{thiscell}."\n";
```

Comment:
there's no need for the stirng concat, `"$config->{thiscell}\n"` should also work, now that `$config` is a hashref.